crates/rust-toolchain.toml

```diff
@@ -1,4 +1,4 @@
 [toolchain]
-channel = "nightly-2024-04-17"
-targets = [ "thumbv7em-none-eabihf" ]
-components = [ "clippy", "rustfmt" ]
+channel = "stable"
```

Comment:
Is there a benefit to using MSRV here? Probably less so for embedded than for other kinds of library crates, but if we do want to test against MSRV then we'll want to pin it somewhere.

Reply:
It still needs `nighlty` for the cargo feature `different-binary-name`, any idea how to get around that one?

Reply:
That's unfortunate. While `different-binary-name` isn't strictly necessary, it does make for a much better overall experience. Let me see if I can think of something.

Reply:
We might be able to do that from `fap-lld`. Let me see how hard that change would be.

Reply:
So while yes, `fap-lld` can rename the file, this is too early in the build process. At this stage the binary is still in `{name}-{hash}` form before being renamed to the final binary.

So I don't think there's any way short of a wrapper script or sticking with nightly.

Reply:
Reverted the `stable` change and switched to `nightly-2024-09-10` instead.